object-construction-checker/tests/mustcall/CrashOnCommon.java

```diff
@@ -0,0 +1,10 @@
+import java.net.*;
+
+class CrashOnCommon {
+//    Object baz = new Object(new Socket());
```

Comment:
Can we add the field declaration test as well and make sure we get an error?

Reply:
https://github.com/kelloggm/object-construction-checker/pull/300/commits/a71831579d604bf70bc86fe342299488b4f80141